saw-remote-api/run_rpc_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pushd $DIR/python
+
+NUM_FAILS=0
+function run_test {
+    "$@"
+    if (( $? != 0 )); then NUM_FAILS=$(($NUM_FAILS+1)); fi
+}
+
+echo "Setting up python environment for remote server clients..."
+poetry install
+
+export SAW_SERVER=$(cabal v2-exec which saw-remote-api)
+if [[ -x "$SAW_SERVER" ]]; then
+  echo "Running saw-remote-api tests..."
+  echo "Using server $SAW_SERVER"
+  run_test poetry run python -m unittest discover tests/saw
```

Comment:
It looks like these tests are run both here and in `build_and_test_docker.sh`. Is this duplication of functionality intended?

Reply:
Yes - the idea is to have the CI check that the server works both when used locally as a binary and when used from a  docker container (previously we only checked the former and we missed issues in the latter). At the moment the tests happen to be the same, but it wouldn't surprise me at all if some tests emerge that are relevant to only one of the other.